tests/lib/metrics_test.py

```diff
@@ -42,11 +47,16 @@ def test_format_dimensions():
 
 
 def test_put_metric_catches_error(mock_session):
-    client = boto3.client("cloudwatch")
-    stubber = Stubber(client)
-
-    stubber.add_client_error("put_metric_data", "InternalServiceError")
-    stubber.activate()
+    client = mock_session.client("cloudwatch")
```

Comment:
big fan of stubber but haven't gotten it working with the mocked session.  This also brings more consistency as these are the only tests using stubber.